media-plugins/mythplugins/mythplugins-30.0.ebuild

```diff
@@ -0,0 +1,167 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 )
+PYTHON_REQ_USE="xml"
+BACKPORTS="03f44039848bd09444ff4baa8dc158bd61454079"
+MY_P=${P%_p*}
+
+inherit eutils python-single-r1 readme.gentoo-r1 vcs-snapshot
+
+DESCRIPTION="Official MythTV plugins"
+HOMEPAGE="https://www.mythtv.org"
+SRC_URI="https://github.com/MythTV/mythtv/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+LICENSE="GPL-2+"
+KEYWORDS="~amd64 ~x86"
+SLOT="0/${PV}"
```

Comment:
I really wonder if subslot is necessary here. What is going to link its binary against libraries in this package?

Reply:
Mythtv and Mythplugins are supposed to be at the same version. Different versions of mythtv use a different mysql database schema. They need to match.

Reply:
then a subslot is not what you want. if the versions are in sync anyway, just make it depend on `=mythtv-${PV}`

Reply:
I remember a time in the past (long before I started maintaining this ebuild) there was a blocking problem in that mythtv would not update because the installed mythplugins blocked it even though mythplugins was also being updated. The workaround was to uninstall mythplugins, upgrade mythtv, then reinstall mythplugins. I think subslotting was an attempt to fix that. I'll remove the subslotting.

Does `RDEPEND=" !<media-tv/mythtv-${PV} ..."` fix that.

Reply:
there is no reason for that happening other than someone excluding mythplugins from upgrade. a blocker is always worse than a versioned dependency as far as dependency resolution goes.

Reply:
I run mythtv [mythbackend as a daemon and mythfrontend/mythplugins as an application on my big screen TV] continuously. 

Does this mean that `DEPEND` `=mythtv-${PV}` will block if the old version of mythtv and mythplugins is running while emerging the new version?

Reply:
You can definitely try the logic in your local overlay ;)

it should work seamlessly. Ie, if you update ebuild for mythtv to 31, people who have mythtvplugins-30 installed wont even see the update, while people who dont have mythtvplugins installed will receive the update. Then when you update mythtvplugins-31 into the tree, everyone will see the update. There are no user stopping blocks.

... unless you for some reason have mythtv-31 in your world file. 

What you're currently doing isnt gonna work in the long run either. If you allow people to mix, say mythtvplugins-31 with mythtv-36, someones gonna hit that. I think using the method @a17r suggested is best. Also to just depend on 31 instead of 31.0, you can depend on mythtv-31*

Reply:
Change to  `DEPEND` `=media-tv/mythtv-${PV}*` to allow patch versions.